.gitignore

```diff
@@ -9,4 +9,5 @@ tectonic-*.tar.gz
 tectonic-license.txt
 terraform.tfstate*
 terraform.tfvars
-/build/
+build/
+modules/azure/vnet/nsg-lb.tf
```

Comment:
Can you do a local ignore of the file so we don't add this to the master .gitignore?